docs/source/api/recipes/recipes.ipynb

```diff
@@ -835,6 +835,73 @@
     "matching_ids = [k for k, v in H.degree().items() if v == d]\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    # Get all of the dates
</pre><p>I would replace this with something like:</p><pre class="ql-syntax" spellcheck="false"># get all the hyperedges corresponding to email sent in the interval between date1 and date2
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Fixed.